src/Confluent.Kafka/Impl/SafeKafkaHandle.cs

```diff
@@ -236,6 +236,10 @@ internal int Flush(int millisecondsTimeout)
         internal int AddBrokers(string brokers)
         {
             ThrowIfHandleClosed();
+            if (brokers == null)
+            {
+                throw new NullReferenceException("List of brokers is required.");
```

Comment:
`ArgumentNullException` would be better here and maybe text: "Argument 'brokers' must not be null". I'm thinking list is 'ambiguous' as is 'required'.

Reply:
Implemented as recommended.